TypeCobol/Compiler/Diagnostics/CrossChecker.cs

```diff
@@ -1031,6 +1031,25 @@ public override bool Visit(Write write)
             return true;
         }
 
+        public override bool Visit(Repository repository)
+        {
+            if (repository.CodeElement is RepositoryParagraph repositoryParagraph && repositoryParagraph.IntrinsicFunctions != null)
```

Comment:
The type checking is useless, please remove

Reply:
Removed